README.md

```diff
@@ -36,7 +36,8 @@ $ source spresense_env.sh
 This script will create an alias `spresense` which should preceed the regular SDK build scripts and Make commands.
 examples
 ```
-SpresenseSDK: $ spresense make
+SpresenseSDK: $ spresense sh -c "cd /spresense/sdk && tools/config.py examples/hello"
```

Comment:
I think it's enough to change it to:
SpresenseSDK: $ spresense tools/config.py examples/hello
SpresenseSDK: $ spresense make

Could you please check if it works?

Reply:
you're right! I'll make the changes